Automattic/jetpack
# Payment Blocks: add button typography panel to payment button

Fixes #25928

#### Changes proposed in this Pull Request:

* We've added support for the Core typography setting to our Jetpack/Button block.

#### Other information:


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Load the changes on a Simple, Atomic and JN site.
* Open or edit a post.
* Add a recurring payments block.
* In the sidebar you should have the typography controls available (Font controls are theme dependant, 22 theme does have these controls)
* The controls should work as expected.
* Save the post and open the post contents, the typography changes should transfer.
